Update /Volunteer (#185)
nycmeshnet/nycmesh.net: content/volunteer/index.md

If you're interested in volunteering with NYC Mesh there's a few ways to get involved:

Join us for our regular weekly Open Mesh Night. This is hosted at our Mesh Room in the basement of BlueStockings Bookstore in the Lower East Side, every Tuesday night at 7:30pm. Open Mesh Night is a great time to meet the community and learn about projects that other members are working on.

There's a weekly thread in [#meetings on Slack](https://slack.nycmesh.net) where you can confirm Open Mesh Night is happening as scheduled. (We sometimes skip holidays or dates when few people are available)

Come talk to us about volunteering opporunities at one of our monthly [Meetups](https://www.meetup.com/nycmesh/). We're super friendly!

You can also join us on [Slack](https://slack.nycmesh.net), our community message board. We use slack to coordinate almost all mesh activities, so joining the Slack is often the first step to getting involved.
There are a number of channels on slack where updates are posted about ongoing projects. You can simply join in on the conversation to get involved!

> See futher down this page for a number of suggested slack channels for new volunteers.

Reviewer: ```suggestion
See further down this page for a number of suggested slack channels for new volunteers.
```

Install coordinators and planners work primarily with our support ticket system to respond to [Join Form](https://nycmesh.net/join) requests that new members submit.
They evaluate the panorama photos that members submit, determine if the new member can connect to the mesh and then pass the install to the Mesh Installers to complete the install.
They also work to answer email support tickets and other questions and enquiries.

**Maintenance Team [#maintenance-team](https://nycmesh.slack.com/messages/maintenace-team)**: 
The maintenance team works to build and maintain the larger hubs and buildings throughout the mesh. We plan install days when a large number (5-12) of volunteers will work together to get a new hub or building online in an afternoon or two.
Upcoming maintenance team projects are scheduled in [#maintenance-tasks](https://nycmesh.slack.com/messages/maintenance-tasks) and maintenance team is used to coordinate and plan these dates.

Joining maintenace team is a great way to get involved with larger buildings, as we always need more help on these big projects!

***
**Network administrator [#monitoring](https://nycmesh.slack.com/messages/monitoring)**:

> Network admistrators deal with deeper level network configuration and plan overall mesh routing and infrastructure. They are often involved with planning, deploying and maintaining new Supernodes, along with helping with configuration for larger hubs.

Reviewer: ```suggestion
Network administrators deal with deeper level network configuration and plan overall mesh routing and infrastructure. They are often involved with planning, deploying and maintaining new Supernodes, along with helping with configuration for larger hubs.
```

Outreach leaders also work to foster colaboration with other community organizations we are in partnerships with. Both finding new organizations to work with and managing our existing relationships.

**Grant writer and sponsorship coordinator [#funding](https://nycmesh.slack.com/messages/funding)**
We need help identifying grants, writing applications and reaching out to sponsors.

**Graphic Designer [#design](https://nycmesh.slack.com/messages/design)**
We have a number of volunteers in #design who create new flyers, posters, letters and other materials for our outreach teams to use. If you are interested in designing posters, t-shirts, stickers or anything else, this is the place to be!
We expecially need people who speak French, Spanish, Mandarin or other languages to help make our materials accessible to non english speakers.

***

***
**Donate Space or Equipment [#general](https://nycmesh.slack.com/messages/general)**

> NYC Mesh accepts donations of working and resonably modern networking equipment and antennas. We decide what to accept on a case by base basis. It's best to reach out in #general with what you have and we can let you know if we can use it.

Reviewer: ```suggestion
NYC Mesh accepts donations of working and reasonably modern networking equipment and antennas. We decide what to accept on a case by case basis. It's best to reach out in #general with what you have and we can let you know if we can use it.
```

> We also welcome donations of space where we can host meetups and events. We are looking to host events in diverse neighborhoods around the city that our further away from our regular Lower East Side meetups. If you have a space to offer or event we should attend, please let us know!

Reviewer: ```suggestion
We also welcome donations of space where we can host meetups and events. We are looking to host events in diverse neighborhoods around the city that are further away from our regular Lower East Side meetups. If you have a space to offer or an event we should attend, please let us know!
```
